src/main/java/frc/lib/tuneables/Tuneables.java

```diff
@@ -0,0 +1,72 @@
+package frc.lib.tuneables;
+
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.function.BiConsumer;
+
+import edu.wpi.first.util.sendable.Sendable;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.lib.tuneables.sendableproperties.TuneableProperty;
+
+public class Tuneables {
```

Comment:
It's a bit weird that the name of this file is Tuenables and there is another file in the name Tuenable, it feels weird and confusing

Reply:
Yeah, I also was annoyed by this but didn't have a good name. Maybe `TuneablesManager`? Have any other ideas?

Reply:
TuneableManager is actually a pretty good name, much better